guides/v2.3/graphql/reference/products.md

```diff
@@ -173,16 +168,10 @@ Attribute | Data type | Description
 `categories` | [CategoryInterface] | The categories assigned to the product. See [categories endpoint]({{ page.baseurl }}/graphql/reference/categories.html) for more information
 `country_of_manufacture` | String | The product's country of origin
 `created_at` | String | Timestamp indicating when the product was created
-`custom_design` | String | A theme that can be applied to the product page
-`custom_design_from` | String | The beginning date when a theme is applied to the product page
-`custom_design_to` | String| The date at which a theme is no longer applied to the product page
-`custom_layout` | String | The name of a custom layout
-`custom_layout_update` | String | XML code that is applied as a layout update to the product page
 `description` | ComplexTextValue | An object that contains detailed information about the product. The object can include simple HTML tags
 `gift_message_available` | String | Indicates whether a gift message is available
 `id` | Int | The ID number assigned to the product
 `image` | ProductImage | An object that contains the URL and label for the main image on the product page
-`is_returnable` | String | Indicates whether the product can be returned. This attribute is defined in the Rma module.
```

Comment:
Keep this attribute. Its origin is listed in the description, and I don't want to create a topic that merely defines one or two attributes

Reply:
**fixed**